src/promptflow-azure/tests/sdk_cli_azure_test/e2etests/test_pfs_azure.py

```diff
@@ -21,8 +21,16 @@ def pfs_op(app: Flask):
     # Hack to import the pfs test utils from the devkit tests
     import sys
 
-    sys.path.append("../promptflow-devkit/tests")
-    from sdk_pfs_test.utils import PFSOperations
+    temp_path = (
+        Path(__file__)
```

Comment:
you can import PROMPTFLOW_PATH

Reply:
from _constants import PROMPTFLOW_ROOT
temp_path = PROMPTFLOW_ROOT / "src/promptflow-devkit/tests/sdk_pfs_test"